money-monkey-web/app/ui/MainSpendDisplay.tsx

```
@@ -1,37 +1,51 @@
+        } satisfies CircleSection)
     );
 
   if (sums == undefined || total == 0) {
     return (
-      <svg className="height-[300px]" width={"100%"} height={"100%"} viewBox={`0 0 ${dim} ${dim}`}>
+      <svg
+        className="height-[300px]"
+        width={"100%"}
+        height={"100%"}
+        viewBox={`0 0 ${dim} ${dim}`}
+      >
```

Biome

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)